src/cordova/apple/xcode/ios/Outline.xcodeproj/xcshareddata/xcschemes/VpnExtension.xcscheme

```diff
@@ -21,30 +21,13 @@
                ReferencedContainer = "container:Outline.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
-         <BuildActionEntry
```

Comment:
Bogus dependency on the Outline target. I think this schema was originally created by copying the one for the app.